sql-gremlin/src/main/java/org/twilmes/sql/gremlin/adapter/converter/ast/nodes/select/GremlinSqlSelectMulti.java

```diff
@@ -203,10 +203,10 @@ protected void runTraversalExecutor(final GraphTraversal<?, ?> graphTraversal,
             graphTraversal = g.E().hasLabel(edgeLabel)
                     .where(__.inV().hasLabel(inVLabel))
                     .where(__.outV().hasLabel(outVLabel));
+            applyHaving(graphTraversal);
```

Comment:
Having isn't really the right word any more, a HAVING should be applied on the results of a group by, vs. a WHERE is applied to the pre-aggregated input.